.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+build
+coverage
```

Comment:
I've a hunch we should ignore `public` too - since that is gatsby's build output.

Reply:
Seems to be useful to add `static` and `.cache` too